pkg/containers/containers.go

```diff
@@ -225,6 +225,9 @@ func getContainerIdFromCgroup(cgroupPath string) (string, string) {
 		case strings.HasPrefix(id, "cri-containerd-"):
 			runtime = "containerd"
 			id = strings.TrimPrefix(id, "cri-containerd-")
+		case strings.Contains(pc, ":cri-containerd:"):
+			runtime = "containerd"
```

Comment:
I think we should start having runtime names under const vars for multiple usage, as it is happening here with `"containerd"`.